src/test/java/org/apache/commons/lang3/math/NumberUtilsTest.java

```diff
@@ -918,7 +918,8 @@ public void testIsParsable() {
         assertFalse(NumberUtils.isParsable("pendro"));
         assertFalse(NumberUtils.isParsable("64, 2"));
         assertFalse(NumberUtils.isParsable("64.2.2"));
-        assertFalse(NumberUtils.isParsable("64."));
+        assertTrue(NumberUtils.isParsable("64."));
```

Comment:
please add a test that 64.. is false

Reply:
@elharo sorry. My understanding of the ticket is that 64. is valid

Reply:
64. is valid but 64.. (two periods) is not

Reply:
you are right. I added a test for it